frontend/src/Commons/Header/Header.style.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+import { screenSizes } from '../../Constants/ScreenSize';
+
+const StyledHeader = styled.header`
+  display: flex;
+  align-items: center;
+  max-width: 100%;
+  max-height: 166px;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  flex-direction: column;
+
+  /* Tablet */
+  @media (min-width: ${screenSizes.tablet}){
+
+  }
+  /* Desktop */
```

Comment:
These comments aren't needed since your Media query object says exactly what it is